source/Settings/set1/frame_01.yml

```diff
@@ -0,0 +1,23 @@
+__version__: 3
```

Comment:
This will be much better in API 2 since all settings for one HDR will be in one file.
I would consider not adding this sample now, instead hold it until API 2 is released.

I plan to setup a API 2 review with CSU early next week, I can explain more then.

Reply:
We have this for C++ and python already. We will update the sample for 2.0 accordingly.